python-avd/pyavd/_cv/workflows/models.py

```diff
@@ -73,6 +73,23 @@ class CVPathfinderMetadata:
     device: CVDevice | None = None
 
 
+@dataclass
+class CVWSBuildConfigValidationError:
+    configlet_name: str | None = None
+    """Name of the Studio-generated configlet which raised validation error(s)."""
+    error_msg: str | None = None
+    """EOS-returned error message."""
+    line_num: int | None = None
+    """Line number of the violating configuration line within the proposed configlet."""
+
+
+@dataclass
+class CVWSBuildError:
```

Comment:
Since this could have warnings as well, we should consider renaming. Maybe also have a flag if we get it from CV if the device failed or succeeded.
```suggestion
class CVWorkspaceBuildResult:
    """Workspace build result for one device."""

```

Reply:
Fixed